standalone/examples.go

```diff
@@ -0,0 +1,135 @@
+package standalone
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/proto"
+	"github.com/mitchellh/mapstructure"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Example model of an example gRPC request
+type Example struct {
+	Name    string  `json:"name"`
+	Service string  `json:"service"`
+	Method  string  `json:"method"`
+	Request Request `json:"request"`
+}
+
+// MetadataPair (name, value) pair
+type MetadataPair struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+// Request gRPC request
+type Request struct {
+	Timeout  time.Duration
```

Comment:
String was working in the previous version. Since we're using custom marshaling/unmarshaling and the code is already implemented and tested I'd prefer to keep it as is.